google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITProtoColumnTest.java

```diff
@@ -21,20 +21,11 @@
 import static org.junit.Assume.assumeFalse;
 
 import com.google.cloud.ByteArray;
-import com.google.cloud.spanner.DatabaseClient;
-import com.google.cloud.spanner.DatabaseId;
-import com.google.cloud.spanner.Key;
-import com.google.cloud.spanner.KeySet;
-import com.google.cloud.spanner.Mutation;
-import com.google.cloud.spanner.ParallelIntegrationTest;
-import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.*;
```

Comment:
Wildcard imports are not allowed according to Google style: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Done. Thanks for the reference.